### DISCUSSION TITLE
“Share via…” for source and binaries

### DISCUSSION THREAD

**Discussion by kim3-sudo** [community/community#15948] [Mobile]
Using the "Share via..." menu for PDFs and other binaries don't actually export the blob object but rather a webpage rendering of the object. It's especially annoying for PDF documents (which could conceivably be added as documentation or as the result of LaTeX compilation).

Feature request to allow saving of binaries and source code files from the GitHub app instead of a webpage rendering of github.com's view of that page.

**Comment by taki-on** [comment]
Thank you very much for your suggestion. I understand it would be especially useful for a repo storing TeX projects. We will discuss the suggestion within the team :)
